Bastion/src/isaac/bastion/manager/BastionBlockManager.java

```diff
@@ -287,145 +280,4 @@ public String infoMessage(boolean dev, Block block, Block clicked, Player player
 
 		return sb.toString();
 	}
-
-	public void handleBlockPlace(BlockPlaceEvent event) {
```

Comment:
So the way this worked before was the events just called methods in the manager and passed the entire event which was kinda dumb and pointless so I moved the logic to the listeners